### PR TITLE
Separate admin controllers

### PR DESCRIPTION
**Pull request by guerler**
This PR separates the admin controllers of the Toolshed and Galaxy. Although this creates some temporary redundancy, this avoids conflicts when we move forward and modernize the Galaxy web app by removing makos and moving/redirecting endpoint operations to the Galaxy api.

### CODE REVIEW

**Comment by guerler** [comment]
@galaxybot test this

**Comment by guerler** [comment]
@galaxybot test this

**Comment by guerler** [comment]
@galaxybot test this

File: lib/galaxy/webapps/galaxy/api/quotas.py [hunk 1]
```diff
@@ -145,3 +144,11 @@ def undelete( self, trans, id, **kwd ):
             return self._undelete_quota( quota )
         except ActionInputError as e:
             raise HTTPBadRequest( detail=str( e ) )
+
+
+def get_quota( trans, id ):
```
Comment (jmchilton) [inline lib/galaxy/webapps/galaxy/api/quotas.py]: I'm probably missing something but I'm a bit confused by this - where does this get called? All the calls to get_quota seem to be on objects not from modules like this.

**Comment by jmchilton** [comment]
This is very nice - I like this separation a lot - really nice direction to refactor things in. Much needed!

File: lib/galaxy/webapps/galaxy/api/quotas.py (hunk 1 above)
Comment (guerler) [reply lib/galaxy/webapps/galaxy/api/quotas.py]: Awesome. Thanks for catching this. I assumed that this is called due to the previous inheritance of the shared admin controller. I removed the method.

**Review by martenson — APPROVED** [review]
smooth refactoring

**Pull request merged**